dev/openchpl_data-model.sql

```diff
@@ -1412,6 +1412,20 @@ CREATE TABLE openchpl.certification_status_event (
 		MATCH FULL ON DELETE RESTRICT ON UPDATE CASCADE
 );
 
+CREATE TABLE openchpl.cures_update_event (
+        cures_update_event_id bigserial NOT NULL,
```

Comment:
Can we just use "id" here? The type bigserial means that postgres made a sequence called "cures_update_event_cures_update_event_id_seq" but if we just named it "id" then the sequence would be called "cures_update_event_id_seq".